### DISCUSSION TITLE
WebUI "Periodic Sync is Not Running"

### DISCUSSION THREAD

**Discussion by timmyv1** [philosowaffle/peloton-to-garmin#627] [Q&A]
I have switched over to running the WebUI v4.1.0.

I have set it up and it works great except it will not sync automatically to Garmin.

On the Home Tab, there is a grey bubble with white writing that says "Periodic syncing is Not Running" at the top of the screen.

On the settings Tab, I have "Automatic Syncing" turned on with check for new workouts every X seconds set to 3600.

It will usually check Peloton for the new workouts each day but I have to manually click "Sync todays workouts" on the Home tab.

I am not using Two Step Verification for Garmin.

Everything works great except for the automatic syncing to Garmin.

Is this operator error?

**Comment by philosowaffle** [comment]
Is there anything interesting in the logs?

**Reply by timmyv1** [reply]
Nothing interesting in the logs that I can see. 

The 4.1.0 log file only shows one sync history and then stops. NO "Sleeping" line or additional uploading sessions.

The 3.6.0 log file (the last version I had before the upgrade to 4.0 and 4.1) shows multiple sync histories and in between each shows "Sleeping for xxx seconds..." based on the xxx time in my config file.

The config file for 4.1.0 is missing a lot of the information that the 3.6.0 config file has. It starts at "Observability" and then everything below that. Nothing above that.

Here is the 4.1.0 log file:
[log20240127.txt](https://github.com/philosowaffle/peloton-to-garmin/files/14188525/log20240127.txt)


Here is the 3.6.0 log file:
[log20240103.txt](https://github.com/philosowaffle/peloton-to-garmin/files/14188515/log20240103.txt)

I like the WebUI interface better than the old WindowsUI. But it is not syncing automatically. I noticed the same results with 4.0.0 but was only running it for a couple of days before 4.1.0 came out.

**Reply by philosowaffle** [reply]
Found the problem and tracking its resolution here: #634
